File: dbtest.js
// dbtest.js
const { Client } = require("pg");

// PostgreSQL bağlantı bilgileri
const client = new Client({
  host: "localhost",     // Lightsail üzerinde DB başka instance ise IP gir
  port: 5432,            // PostgreSQL varsayılan port
  user: "postgres",      // kendi kullanıcı adın
  password: "s1vc10n",     // kendi şifren
  database: "kampdefterim",    // kendi veritabanın
});

async function connectDB() {
  try {
    await client.connect();
    console.log("Bağlantı başarılı 🚀");
  } catch (err) {
    console.error("Bağlantı hatası ❌", err);
  } finally {
    await client.end();
  }
}

connectDB();

File: valilik-uygulaması/db_test.py
import psycopg2

DB_CONFIG = {
    'host': 'localhost',
    'database': 'kampdefterim',
    'user': 'postgres',
    'password': '"s1vc10n',
    'port': '5432'
}

try:
    conn = psycopg2.connect(**DB_CONFIG)
    print("✅ PostgreSQL bağlantısı başarılı!")
    conn.close()
except Exception as e:
    print(f"❌ Bağlantı hatası: {e}")